packages/site/src/layout/SearchBox.tsx

```diff
@@ -32,6 +32,16 @@ export const SearchBox = ({
   const ref = useRef<InputTextRef>(null);
   const [search, setSearch] = useState("");
 
+  const filteredContentList = useMemo(() => {
```

Comment:
We should come back and improve this so you don't have to copy and paste every time you want to add a new section to the search box

Reply:
Good catch - do you mind creating a ticket for that and we can return to this later on?